vignettes/susceptibility_matrices.Rmd

```diff
@@ -0,0 +1,390 @@
+  column_to_rownames(var = "age_group") %>%
+  as.matrix()
+
+susceptibility
+
+# demography-susceptibility distribution matrix
+p_susceptibility <- tibble(
+  age_group = c("[0,5)", "[5,18)", "[18,40)", "[40,65)", "65+"),
+  immunised = c(0.4, 0.4, 0.4, 0.4, 0.4)
+  ) %>%
+  mutate(unimmunised = 1 - immunised) %>%
+  column_to_rownames(var = "age_group") %>%
+  as.matrix()
+
+p_susceptibility
+```
+
+::: {.alert .alert-warning}
+**Note** that because there are two susceptibility groups, the susceptibility matrix has two columns. The corresponding **demography-susceptibility distribution matrix** must also have two columns!
+
+$$
+4 \; demo. \; groups \left.\vphantom{\begin{matrix}
+s_{1,1} & s_{1,2} \\
+s_{2,1} & s_{2,2} \\
+s_{3,1} & s_{3,2} \\
+s_{4,1} & s_{4,2} 
+\end{matrix}}\right\{
+\overbrace{
+\begin{matrix}
+s_{1,1} & s_{1,2} \\
+s_{2,1} & s_{2,2} \\
+s_{3,1} & s_{3,2} \\
+s_{4,1} & s_{4,2} 
+\end{matrix}}^{\displaystyle 2 \; susceptibility \; groups}
+\quad
+4 \; demo. \; groups \left.\vphantom{\begin{matrix}
+p_{1,1} & p_{1,2} \\
+p_{2,1} & p_{2,2} \\
+p_{3,1} & p_{3,2} \\
+p_{4,1} & p_{4,2} 
+\end{matrix}}\right\{
+\overbrace{
+\begin{matrix}
+p_{1,1} & p_{1,2} \\
+p_{2,1} & p_{2,2} \\
+p_{3,1} & p_{3,2} \\
+p_{4,1} & p_{4,2} 
+\end{matrix}}^{\displaystyle 2 \; susceptibility \; groups}
+$$
+:::
+
+### Case 4: Within groups in different proportion
+
+In the population,
+there is different
+susceptibility to the infection:
+
+- between individuals of different age groups
+from 20% (infants) to 100% (65+), and
+
+- within individuals of the same age group
+due the immunization effect of 25%.
+
+Also the
+immunization uptake rate
+is in different proportions
+for each of the age groups:
+immunization increases with age
+from 20% (infants) to 90% (65+)
+
+``` {r}
+immunization_effect <- 0.25
+
+# susceptibility matrix
+susceptibility <- tibble(
+  age_group = c("[0,5)", "[5,18)", "[18,40)", "[40,65)", "65+"),
+  unimmunised = c(0.2, 0.5, 0.6, 0.9, 1.0)
+  ) %>%
+  mutate(immunised = unimmunised * (1 - immunization_effect)) %>%
+  column_to_rownames(var = "age_group") %>%
+  as.matrix()
+
+susceptibility
+
+# demography-susceptibility distribution matrix
+p_susceptibility <- tibble(
+  age_group = c("[0,5)", "[5,18)", "[18,40)", "[40,65)", "65+"),
+  immunised = c(0.2, 0.4, 0.6, 0.7, 0.9)
+  ) %>%
+  mutate(unimmunised = 1 - immunised) %>%
+  column_to_rownames(var = "age_group") %>%
+  as.matrix()
+
+p_susceptibility
+```
+
+### Case 5: Within three groups
+
+In the population,
+there is different
+susceptibility to the infection:
+
+- between individuals of different age groups
+from 20% (infants) to 100% (65+), and
+
+- within individuals of the same age group
+due the immunization effect of 25%
+to the 40% of each of the age groups.
+
+Additionally,
+10% of individuals
+in each of the age groups
+have
+100% susceptibility,
+due to no immunization
+or not exposed
+to similar pathogens previously.
```

Comment:
Could these sentence be in a single line?

Reply:
Fixed in https://github.com/epiverse-trace/finalsize/pull/144/commits/e08191cc1b69e7bf9fc9895eb69093720baff1db